utils/data_utils.py

```diff
@@ -125,7 +139,15 @@ def get_summary(df_exposure, df_adm, adm_level):
     )
     people_exposed_formatted = "{:,}".format(people_exposed)
 
-    return (
-        name,
-        f"{people_exposed_formatted} people exposed to flooding as of {max_date}.",
+    # TODO: Remove need for mapping
+    tercile_label = {-1: "below average", 0: "average", 1: "above average"}
+
+    summary_text = dcc.Markdown(
+        f"""
+        **{people_exposed_formatted}** people exposed to flooding as of **{max_date}**.
+
+        This is **{tercile_label[tercile]}** for this time of year.
```

Comment:
Another tiny thing- we may want to say "for this day of the year" instead of this time of year. Just a bit more precise (and I think it's ok to say "day" instead of 7-day rolling average, since we already specify in the methodology that it's a 7-day rolling average).